src/function/memo.rs

```diff
@@ -217,12 +216,12 @@ impl<V> Memo<V> {
 
     /// Mark memo as having been verified in the `revision_now`, which should
     /// be the current revision.
+    #[inline]
```

Comment:
It would be great to have a comment here mentioning that it's the caller's responsibility to update `accumulated` if their accumulated values have changed since  the query ran last